genume/registry/child.py

```diff
@@ -0,0 +1,88 @@
+from shlex import split as cmdsplit
+import subprocess as sbpr
+import logging as log
+import time
```

Comment:
The `time.time_ns` is not supported in all python 3.*

You should add something like:
```diff
import time

+if hasattr(time, "time_ns"):
+    def time_ns():
+        return time.time_ns()
+else:
+    def time_ns():
+        return time.time() * 1000000
+

class ChildHandler:
```

and then call `time_ns()` instead of `time.time_ns()`.

You can also add it at the utils module if you want

Reply:
Got it!